test/language/expressions/arrow-function/arrow/apply_2.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply - 'this' is lexically bound
+es6id: 14.2
+---*/
+
+var b = {
+    method: () => this
+ }

```

Comment:
> This test here could focus on the four asserts using my sentinel suggestion above and not worry about an enclosing object literal.

@ianwjhalliday I believe that's more than sufficiently covered here:   https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js ?